lib/vanagon/component.rb

```diff
@@ -215,7 +217,7 @@ def get_source(workdir) # rubocop:disable Metrics/AbcSize
         @dirname = './'
 
         # If there is no source, there is nothing to do to extract
-        @extract_with = ': no source, so nothing to extract'
+        extract_with << ': no source, so nothing to extract'
```

Comment:
I wonder if I could ditch this line entirely. An empty `Array` is a safe array, and it'd cut down on chatter and verbosity in the Makefile.

![man_presses_fingers_together_thinking_something_cg4p41662c](https://cloud.githubusercontent.com/assets/344926/23968173/f5f8e66e-097e-11e7-9bf9-137fcc25db7d.jpg)